SU-SOE/stanford_soe_helper
# SOE-2763: Added homepage CSS

# READY FOR REVIEW

# Summary
Two Column Tweaks:
- ADDRESS THE HOMEPAGE & PARAGRAPH 2-COLUMN STYLE
- ADDRESS THE WYSIWYG 2-COLUMN STYLE, SO THAT IT LOOKS LIKE A SINGLE COLUMN ON LEGACY CONTENT, COPY THESE STYLES FOR THE NEW "CALLOUT" WYSIWYG STYLE

# Needed By (Date)
- Sprint end

# Criticality
- Fixed on prod, need to put into code.

# Steps to Test
- Switch to this branch
- Navigate to the homepage
- Verify you see changes on the two column block on the homepage
- Navigate to a magazine article that formerly had a two column block and verify it is a one column block (/magazine/article/why-are-some-breast-cancers-more-likely-spread-others)

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2763

## Related PRs
## More Information
Original code is here:
https://github.com/SU-SOE/SoE-CSS/blob/master/Two%20Columns%20tweaks

## Folks to notify
@kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
